README.md

```diff
@@ -54,7 +54,7 @@ To install this gem onto your local machine, run `bundle exec rake install`. To
 ### Docker
 
 ```
-docker run -it -d -v $(pwd):/src ruby /bin/bash
```

Comment:
Is there a reason to be explicit here? As a library, we expect it to work with all supported MRI versions. If we do want to be explicit (and maybe even if we don't), we should update `.ruby-version`, which is 2.2.5.